app/models/api/openid_connect/id_token.rb

```diff
@@ -0,0 +1,52 @@
+module Api
+  module OpenidConnect
+    class IdToken < ActiveRecord::Base
+      belongs_to :authorization
+
+      before_validation :setup, on: :create
+
+      default_scope { where("expires_at >= ?", Time.zone.now.utc) }
+
+      def setup
+        self.expires_at = 30.minutes.from_now
+      end
```

Comment:
No, I suppose I should create a background job that runs every day? Just like https://github.com/diaspora/diaspora/pull/6147/files